ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/CurrentGroupParametersService.java

```diff
@@ -0,0 +1,26 @@
+package net.corda.v5.ledger.utxo;
+
+import net.corda.v5.membership.GroupParameters;
+import org.jetbrains.annotations.NotNull;
+
+import java.security.PublicKey;
+import java.util.List;
+
+/**
+ * {@link CurrentGroupParametersService} allows flows to access {@link GroupParameters} for further verification,
+ * for example, accessing to check if issued money is from authoritative party in flows.
+ */
+public interface CurrentGroupParametersService {
+
+    /**
+     * Returns the point in time {@link GroupParameters}
+     */
+    @NotNull
+    GroupParameters getCurrentGroupParameters();
+
+    /**
+     * Returns the list of {@link PublicKey} of MGM.
+     */
+    @NotNull
+    List<PublicKey> getMgmKeys();
```

Comment:
I think this one does not need to be publicly accessible.
You can remove it from here.